confidential-datahub/kms/src/error.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use thiserror::Error;
+
+pub type Result<T> = std::result::Result<T, Error>;
+
+#[derive(Error, Debug)]
+pub enum Error {
+    #[error("Aliyun KMS error: {0}")]
+    AliyunKmsError(String),
```

Comment:
Should this be behind feature flag? I guess it doesn't really matter.

Reply:
makes sense, let me fix it then